app.json

```diff
@@ -11,7 +11,7 @@
     "formation": {
         "web": {
             "quantity": 1,
-            "size": "hobby"
+            "size": "basic"
```

Comment:
Apparently Heroku dropped the hobby plan, so we need to use basic instead (review apps are broken at the moment on all PRs)